plugins/modules/inventory_intent.py

```diff
@@ -2828,9 +2907,9 @@ def verify_diff_merged(self, config):
             if device_role_flag:
                 self.status = "success"
                 msg = "Device roles updated and verified successfully."
-                self.log(msg)
+                self.log(msg, "INFO")
             else:
-                self.log("Playbook parameter does not match with Cisco Catalyst Center, meaning update device role task not executed successfully.")
+                self.log("Playbook parameter does not match with Cisco Catalyst Center, meaning update device role task not executed successfully.", "INFO")
```

Comment:
self.log("Mismatch between playbook parameter '{0}' and Cisco Catalyst Center detected, indicating the device role update task may not have executed successfully.", "INFO")

Reply:
added.